tailwindcss/mod.ts

```diff
@@ -0,0 +1,38 @@
+import type { Page, Site } from "lume/core.ts";
+import { merge } from "lume/core/utils.ts";
+
+import { addCssLink2Head, getTwBinFullPath } from "./deps.ts";
+
+export interface Options {
+  version: string;
+  dir: string;
+}
+
+// Default options
+export const defaults: Options = {
+  version: "3.1.8",
+  dir: "./_bin",
+};
+
+export default function tailwindcss(userOptions?: Partial<Options>) {
+  const { version, dir } = merge(defaults, userOptions);
+
+  return (site: Site) => {
+    site.process([".html"], (page: Page) => addCssLink2Head(page));
+
+    site.addEventListener("afterBuild", async () => {
+      const tailwindBin = await getTwBinFullPath(version, dir);
+      const process = Deno.run({
+        cmd: [
+          tailwindBin,
+          "-i",
+          "./_site/css/main.css",
```

Comment:
The demo doesn't work because the input file doesn't exist. This argument is optional and should be configurable. For example:

```js
site.use(tailwind({
  input: "/css/main.css"
});
```

If no input is specified, this argument is not used.